docs/docs/misc/migration_notes.md

```diff
@@ -6,12 +6,74 @@ keywords: [sandbox, cli, aztec, notes, migration, updating, upgrading]
 
 Aztec is in full-speed development. Literally every version breaks compatibility with the previous ones. This page attempts to target errors and difficulties you might encounter when upgrading, and how to resolve them.
 
+## TBD
```

Comment:
Should this ^ be removed?

Reply:
You can add `0.24.0` if you think you can get it merged today. Or `0.25.0` if not. This will stay as tech debt otherwise. You can see the next release numbers here - https://github.com/AztecProtocol/aztec-packages/pull/4491 (search for "release" in open PRs)